code/controllers/subsystem/vote.dm

```diff
@@ -176,10 +176,14 @@ SUBSYSTEM_DEF(vote)
 		return FALSE
 	// If user has already voted, remove their specific vote
 	if(usr.ckey in voted)
+		if(usr.client.patreon.has_access(SENIOR_AGENT_RANK))	//Patreons get double mapvote
```

Comment:
The comment is a half-truth. This seems to apply to any vote, be that Restart, Custom, Transfer, Map or Gamemode (yeah I know).

Is this intended behavior?

Reply:
Yes; It is. the comment is from an old version of it.